src/react/components/common/tagInput/tagInput.tsx

```diff
@@ -492,6 +489,15 @@ export class TagInput extends React.Component<ITagInputProps, ITagInputState> {
         }
     }
 
+    private getTagCatagory = (tagType: string) => {
+        switch (tagType) {
+            case FieldType.SelectionMark:
+                return "checkbox";
+            default:
+                return "text";
+        }
+    }
+
```

Comment:
here we can do even shorter in one line: 
    private getTagCategory = (tagType: string) => tagType === FieldType.SelectionMark ? "checkbox" : "text";

Reply:
and misspelled 'category'